editor/store.js

```diff
@@ -26,7 +27,11 @@ const GUTENBERG_PREFERENCES_KEY = `GUTENBERG_PREFERENCES_${ window.userSettings.
 function createReduxStore() {
 	const enhancers = [
 		applyMiddleware( multi, refx( effects ) ),
-		storePersist( 'preferences', GUTENBERG_PREFERENCES_KEY ),
+		storePersist( {
+			reducerKey: 'preferences',
+			storageKey: GUTENBERG_PREFERENCES_KEY,
+			defaults: STORE_DEFAULTS.preferences,
```

Comment:
It might be a good idea to simplify `STORE_DEFAULTS.preferences` to `PREFERENCES_DEFAULTS` as it is the only item there.